### PR TITLE
[zh-cn]: update the translation of ArrayBuffer.@@species

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#16958

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/@@species/index.md [hunk 1]
```diff
@@ -1,35 +1,63 @@
...
+title: ArrayBuffer[@@species]
 slug: Web/JavaScript/Reference/Global_Objects/ArrayBuffer/@@species
 ---
 
 {{JSRef}}
 
-该 **`ArrayBuffer[@@species]`** 访问器属性会返回 `ArrayBuffer` 构造器。
+**`ArrayBuffer[@@species]`** 静态访问器属性返回用于构造数组缓冲区方法返回值的构造函数。
+
+> **警告：** `@@species` 的存在允许执行任意代码，并可能产生安全漏洞。这也使某些优化变得更加困难。引擎实现者正在[调查是否删除此功能](https://github.com/tc39/proposal-rm-builtin-subclassing)。如果可能的话，避免依赖它。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/@@species/index.md:10]:
```suggestion
> **警告：** `@@species` 的存在允许任意代码执行，并可能产生安全漏洞。这也使某些优化变得更加困难。引擎实现者正在[调查是否删除此特性](https://github.com/tc39/proposal-rm-builtin-subclassing)。如果可能的话，请避免依赖它。
```

File: files/zh-cn/web/javascript/reference/global_objects/arraybuffer/@@species/index.md [hunk 2]
```diff
@@ -1,35 +1,63 @@
...
+```
+
+由于这种多态实现，派生子类的 `@species` 也将默认返回构造函数本身。
+
+```js
+class SubArrayBuffer extends ArrayBuffer {}
+SubArrayBuffer[Symbol.species] === SubArrayBuffer; // true
+```
+
+当调用不会修改现有对象，而是返回一个新的数组缓冲区实例数组缓冲区方法（例如，[`slice()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice)）时，对象的 `constructor[@@species]` 将被访问。返回的构造函数将用于该构造数组缓冲区方法的返回值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/arraybuffer/@@species/index.md:42]:
```suggestion
当调用不会修改现有对象，而是返回一个新的数组缓冲区实例数组缓冲区方法（例如，[`slice()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice)）时，对象的 `constructor[@@species]` 将被访问。返回的构造函数将用于构造该数组缓冲区方法的返回值。
```

**Review by yin1999 — COMMENTED** [review]
Almost looks good. Just two nits

**Comment by yin1999** [comment]
You haven't resolved those comments, is there any other thoughs on it?

**Comment by JasonLamv-t** [comment]
> You haven't resolved those comments, is there any other thoughs on it?

Sorry about that, I wrongly thought that I had already made these changes commit.

**Review by yin1999 — APPROVED** [review]
Thanks @JasonLamv-t :+1:

**Pull request merged**